src/tree/registries/gitLab/gitLabRegistryProvider.ts

```diff
@@ -0,0 +1,21 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { RegistryApi } from "../all/RegistryApi";
+import { IRegistryProvider } from "../IRegistryProvider";
+import { GitLabAccountTreeItem } from "./GitLabAccountTreeItem";
+
+export const gitLabRegistryProvider: IRegistryProvider = {
+    label: "GitLab",
+    id: 'gitLab',
+    api: RegistryApi.GitLabV4,
+    logInOptions: {
+        wizardTitle: 'Log In to GitLab',
```

Comment:
nit: I think in wouldn't be capitalized in this case.  But what do you think about using "Sign in" to be consistent with Azure Accounts?

Reply:
This was modeled after the existing Docker Hub support which uses "Log In". I don't have a strong preference

Reply:
Yeah same, just thought I'd mention it